README.md

```diff
@@ -95,10 +97,37 @@ After your Docker container successfully builds and is running locally you can s
 3. Inside the `launch.json` file, set the `configuration->service->serviceAccountName` attribute to the service account email you just created.
 
 ## Deploying your own instance with terraform
-To run the pipeline with terraform, create your own `terraform.tfvars` file in the same directory as the other terraform files. For each variable declared in `prototype_variables.tf` that doesn't have a default, add your own for testing. Typically your own variables should be unique and can just be prefixed with your name or ldap. There are some that have specific requirements like project ids, code paths, and image paths.
-
-Currently the setup deploys both a data ingestion cloud funtion and a data ingestion cloud run instance. These are duplicates of each other. Once we get the cloud run one fully working we will delete the cloud fuction, but for now if you want to not have a duplicate while developing you can just comment out the setup for whichever one you don't want to use in `prototype.tf`
-
-Note that currently terraform doesn't diff the contents of the functions/cloud run service, so to get them to redeploy you can either
-1. Call `terraform destroy` every time before `terraform apply`, or
-2. Use [`terraform taint`](https://www.terraform.io/docs/commands/taint.html) to mark a resource as requiring redeploy
+Before deploying, make sure you have installed Terraform and a Docker client (e.g. Docker Desktop). See [One time setup](#one-time-setup) above.
+1. Create your own `terraform.tfvars` file in the same directory as the other terraform files. For each variable declared in `prototype_variables.tf` that doesn't have a default, add your own for testing. Typically your own variables should be unique and can just be prefixed with your name or ldap. There are some that have specific requirements like project ids, code paths, and image paths.
+2. Configure docker to use credentials through gcloud.  
+```gcloud auth configure-docker```
+3. On the command line, navigate to your project directory.  
+```cd path/to/your/project```
+4. Build and push your Docker images to Google Container Registry. Do this for both the run_ingestion and run_gcs_to_bq services.  
+   ```bash
+   # Build the image locally
+   docker build -t gcr.io/<project-id>/<your-image-name> run_ingestion/
```

Comment:
Can we add that your-image-name is something you make up and not something you have to find (like the project ID)